release-1.9/release-1.9.md

```diff
@@ -1,104 +1,127 @@
+
+The 1.9 release cycle begins on Monday, October 2, 2017.
+
+* [Release Team](http://bit.ly/k8s19-team)
+* [Meeting Minutes](http://bit.ly/k8s19-minutes)
+* [Zoom](http://bit.ly/k8s19-zoom)
+* [Slack](https://kubernetes.slack.com/messages/sig-release/)
+* [Forum](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+* [Feature Tracking Sheet](http://bit.ly/k8s19-features)
+* [Milestone Process](https://github.com/kubernetes/community/blob/master/contributors/devel/release/issues.md)
+
+## Notes
+
+* There are a number of holidays plus KubeCon during Q4, so this release will
+  need to have a reduced scope in order to meet the tight schedule.
+* Features that don't have complete code and tests by [Code Freeze](#code-freeze)
+  **may be disabled by the release team** before cutting the first beta.
+* The release team will escalate [release-master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking)
+  failures to SIGs throughout the cycle, not just near release cuts.
+* Key deliverables (e.g. release cuts) tend to be scheduled on Wednesdays
+  to maintain context while ramping up and then responding to any problems.
+
+## Timeline
+
+* Mon Oct 2: Start of release cycle
+* **Fri Oct 27: [Feature Freeze](#feature-freeze)**
+  * All features must have tracking issues.
+* Wed Nov 1: v1.9.0-alpha.2
+* Wed Nov 15: v1.9.0-alpha.3
+* Thu Nov 16: v1.9.0-beta.0
+  * Create `release-1.9` branch and start daily `branchff`.
+  * Start setting up branch CI.
+* **Mon Nov 20: [Code Slush](#code-slush)**
+  * All PRs must be approved for the milestone to merge.
+* **Wed Nov 22: [Code Freeze](#code-freeze)**
+  * All features must be code-complete (*including tests*) and have docs PRs open.
+  * Only release-blocking bug fixes allowed after this point.
+* **Mon Nov 27: [Pruning](#pruning)**
+  * The release team may begin **disabling incomplete features** unless they've
+    been granted [exceptions](#exceptions).
+* Wed Nov 29: v1.9.0-beta.1
+  * Begin manual downgrade testing.
+* Fri Dec 1: Docs Deadline
+  * All docs PRs should be ready for review.
+* Wed Dec 6: v1.9.0-beta.2 (week of KubeCon)
+* **Mon Dec 11: End of Code Freeze**
+  * Perform final `branchff`.
+  * The `master` branch reopens for work targeting v1.10.
+  * PRs for v1.9.0 must now be cherry-picked to the `release-1.9` branch.
+* **Wed Dec 13: v1.9.0**
+* Thu Dec 14: v1.10.0-alpha.1
+* Thu Dec 21: Release retrospective (in community meeting slot)
+
+## Details
+
+### Feature Freeze
+
+All features going into the release must have an associated issue in the
+[features repo](https://github.com/kubernetes/features) by **Fri Oct 27**.
+
+SIG PM will then review features and work with other SIGs to draft release notes
+and themes.
+
+### Code Slush
+
+Starting on **Mon Nov 20**, only PRs that are [approved for the milestone](https://github.com/kubernetes/community/blob/master/contributors/devel/release/issues.md)
+will be allowed to merge into the `master` branch.
```

Comment:
The link refers to issues, not PR's.  Do we disallow PR's that don't reference issues? (This is something we can't construct a github query for)

In github query terms we have historically used the following GitHub Query
`milestone:v1.9` in addition to the usual `label:'cncf-cla:yes' label:approved label:lgtm`

How do we take all the label requirements we've put on issues and tie them to PR's?

Reply:
Note for anyone following along: We moved this discussion to https://github.com/kubernetes/sig-release/issues/26.